src/index.html

```diff
@@ -10,13 +10,64 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>MOYO/Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav-logo">
+        <a href="homepage">
+          <img
+            src="images\logo.png"
+            alt="MOYO-logo"
+          />
```

Comment:
Use forward slashes (`/`) for paths instead of backslashes (`\`) to ensure compatibility across different operating systems.